security.rst

```diff
@@ -548,29 +545,29 @@ You must enable this using the ``login_throttling`` setting:
     .. code-block:: php
 
         // config/packages/security.php
-        $container->loadFromExtension('security', [
-            'enable_authenticator_manager' => true,
+        use Symfony\Config\SecurityConfig;
 
-            'firewalls' => [
-                // ...
+        return static function (SecurityConfig $security) {
+            $security->enableAuthenticatorManager(true);
 
-                'main' => [
-                    // by default, the feature allows 5 login attempts per minute
-                    'login_throttling' => null,
+            // ...
+            $mainFirewall = $security->firewall('main');
 
-                    // configure the maximum login attempts (per minute)
-                    'login_throttling' => [
-                        'max_attempts' => 3,
-                    ],
+            // by default, the feature allows 5 login attempts per minute
+            $mainFirewall
+                ->loginThrottling(null);
```

Comment:
Im not sure this is correct. I need to confirm and possibly make a PR to symfony/symfony to fix int

Reply:
I misunderstood this config fist. I though this config would disable login throttling. 

```yaml
security:
    enable_authenticator_manager: true

    firewalls:
        main:
            pattern: /
            login_throttling: null
```

I was wrong. It actually enables it.. The PR is correct.